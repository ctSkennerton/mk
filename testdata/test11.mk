# TEST_MAIN is set by the test harness.
# Test that mkfile variables are expanded in backquote substitution.

hello = one two three
deps = `echo $hello`

test2.mk.o:
	process $deps
