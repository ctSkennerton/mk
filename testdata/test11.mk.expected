test2.mk.o: process one two three
